source/Xunit.Gherkin.Quick/CoreModel/FeatureFile/IFeatureFileRepository.cs

```diff
@@ -1,10 +1,11 @@
-﻿using System.Collections.Generic;
-
+﻿using System.Collections.Generic;
+
 namespace Xunit.Gherkin.Quick
 {
     internal interface IFeatureFileRepository
     {
-        FeatureFile GetByFilePath(string filePath);
-        List<string> GetFeatureFilePaths();
+        FeatureFile GetByFilePath(string filePath);
+        List<string> GetFeatureFilePaths();
+        List<string> FindFilesByPattern(string pattern);
```

Comment:
This does not belong here conceptually. This is a feature file repository not a file path repository. What are you trying to do with it?